src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -115,6 +116,33 @@ public function denormalize($data, $class, $format = null, array $context = arra
                 $setter = 'set'.ucfirst($attribute);
 
                 if (in_array($setter, $classMethods)) {
+
+                    if ($this->propertyInfoExtractor) {
```

Comment:
Should moved in a `protected` method of `AbstractNormalizer` to be usable in all normalizers (including `ObjectNormalizer`).